nautobot/core/templates/generic/object_detail.html

```diff
@@ -85,23 +90,65 @@ <h1>{% block title %}{{ object }}{% endblock %}</h1>
-            {% include 'inc/relationships_panel.html' %}
-            {% include 'extras/inc/tags_panel.html' %}
-            {% plugin_left_page object %}
-        </div>
-        <div class="col-md-6">
-            {% block content_right_page %}{% endblock content_right_page %}
-            {% plugin_right_page object %}
+    <div class="tab-content">
+        <div id="main" role="tabpanel" class="tab-pane {% if not active_tab and not request.GET.tab or request.GET.tab == "main" %}active{% else %}fade{% endif %}">
+            <div class="row">
+                <div class="col-md-6">
+                    {% block content_left_page %}{% endblock content_left_page %}
+                    {% include 'inc/custom_fields_panel.html' %}
+                    {% include 'inc/relationships_panel.html' %}
+                    {% include 'extras/inc/tags_panel.html' %}
+                    {% plugin_left_page object %}
+                </div>
+                <div class="col-md-6">
+                    {% block content_right_page %}{% endblock content_right_page %}
+                    {% plugin_right_page object %}
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-md-12">
+                    {% block content_full_width_page %}{% endblock content_full_width_page %}
+                    {% plugin_full_width_page object %}
+                </div>
+            </div>
         </div>
-    </div>
-    <div class="row">
-        <div class="col-md-12">
-            {% block content_full_width_page %}{% endblock content_full_width_page %}
-            {% plugin_full_width_page object %}
+        <div id="advance" role="tabpanel" class="tab-pane {% if request.GET.tab == 'advance' %}active{% else %}fade{% endif %}">
+            <div class="row">
+                <div class="col-md-6">
+                    {% include 'inc/object_details_panel.html' %}
+                </div>
+                <div class="col-md-6">
+                    {% block advance_content_right_page %}{% endblock advance_content_right_page %}
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-md-12">
+                    {% block advance_content_full_width_page %}{% endblock advance_content_full_width_page %}
+                </div>
+            </div>
         </div>
     </div>
 {% endblock content %}
+
+
+{% block javascript %}
+<script src="{% static 'clipboard.js/clipboard-2.0.6.min.js' %}"
+    onerror="window.location='{% url 'media_failure' %}?filename=clipboard.js/clipboard-2.0.6.min.js'">
+</script>
+<script>
+    url = window.location.href.split("?")[0];
+    window.history.pushState({}, document.title, url);
+    var clipboard = new ClipboardJS('.btn');
+
+    clipboard.on('success', function (e) {});
+
+    clipboard.on('error', function (e) {});
+
+    function switch_tab(tab_href) {
+        if (window.location.href.includes("changelog")) {
```

Comment:
Probably need logic here to account for other `extra_nav_tabs` besides the changelog tab. Please test with the Prefix and VLAN object detail pages as those both make use of `extra_nav_tabs`.